src/gitlab-ci.coffee

```diff
@@ -132,6 +135,47 @@ module.exports = (robot) ->
             return
 
         return "#{major}.#{minor}.#{dev}"
+    triggerBuildPost = (res, updateRichmond, quickly, branch, buildTagPrefix) ->
+        triggerURL = 'https://agra.sdelements.com/api/v4/projects/33/trigger/pipeline/'
```

Comment:
The URL is set in the start script as an environment variable. So we can likely add the URL for gitlab in that file and read the value in the code instead of hard coding it.